common/src/View/InputEvent.cpp

```diff
@@ -193,6 +193,12 @@ std::ostream& operator<<(std::ostream& lhs, const MouseEvent::WheelAxis& rhs)
   case MouseEvent::WheelAxis::Vertical:
     lhs << "Vertical";
     break;
+  case MouseEvent::WheelAxis::Pinch:
```

Comment:
It feels not quite right to introduce these fake wheel axes. Maybe we could rename the enum to something more generic like `GestureType`?

Reply:
Or is it common practice to treat these events as if they were associated with virtual "axes"?

Reply:
There's _some_ precedent in HTML/DOM: The [`deltaZ`](https://developer.mozilla.org/en-US/docs/Web/API/WheelEvent/deltaZ) axis of `WheelEvent` is used for pinching gestures. (I don't think rotation trackpad gestures are supported on the web, at least with the standardized APIs)

If you feel like it's a hack, I can introduce a separate event type, distinct from scroll specifically for them. (But we'd probably still consolidate them in `InputState`, correct?)